spec/futurist/forking_resolution_strategy_spec.rb

```diff
@@ -122,14 +120,12 @@ def ready_monitor_initializer(_process_id)
   end
 
   it "is not ready when the process has not yet exited" do
-    def not_ready_monitor_initializer(_process_id)
-      stub_monitor(ready: false)
-    end
-    not_ready_monitor_constructor = method(:not_ready_monitor_initializer)
     allow_any_instance_of(Futurist::ForkingResolutionStrategy).
       to receive(:start_promise_evaluation)
     strategy = Futurist::ForkingResolutionStrategy.new(
-      process_monitor_constructor: not_ready_monitor_constructor,
+      process_monitor_constructor: ->(_process_id) {
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Use the `lambda` method for multi-line lambdas.